qlib/contrib/model/catboost_model.py

```diff
@@ -12,18 +15,32 @@
 from ...data.dataset.handler import DataHandlerLP
 from ...model.interpret.base import FeatureInt
 from ...data.dataset.weight import Reweighter
+from ...log import get_module_logger
+
+working_dir = os.getcwd()
+target_dir = os.path.join(os.path.relpath(working_dir), "tensor_board")
+target_dir += "/runs/catboost/"
+sys.path.insert(0, target_dir)
 
 
 class CatBoostModel(Model, FeatureInt):
     """CatBoost Model"""
 
-    def __init__(self, loss="RMSE", **kwargs):
+    def __init__(self, loss="RMSE", tensorboard=False, tensorboard_name="", **kwargs):
```

Comment:
Do you think merging `tensorboard=False, tensorboard_name=""` to a single one will make the code more concise?
For example
````Python
tensorboard_name: Optional[str] = None
````
`None` indicates that tensorboard is not enabled.